.github/pull_request_template.md

```diff
@@ -0,0 +1,26 @@
+## Status
+- Done, In progress, etc
+
+## GitHub Issues addressed (`#IssueNumber`)
+- This PR closes 
+
+## What I did
+- 
+
+## Screenshots
+- Before
+- After
+
+## Tests
+- A brief explanation of tests done/written or how reviewers can test your work
+
+## Questions/Discussions/Notes
+- 
+
+## Merging the PR
+- Who should merge this PR?
+  - [ ] I will merge it myself
+  - [ ] The last reviewer to approve can merge it
+    - Squash-merge?
```

Comment:
I would add a note explaining what squash-merge is/does

Reply:
Clarified in 10e5f36ef30c38b4accdc8132345403c4acde66b